### PR TITLE
Doc fixes

### PR DESCRIPTION
**Pull request by nikolaswise**

### CODE REVIEW

File: docs/source/documentation/components/sample-code/_images.html [hunk 1]
```diff
@@ -1,17 +1,17 @@
...
+             {{relativePath}}assets/img/image-480-md.jpg   480w,
+             {{relativePath}}assets/img/image-240-sm.jpg   240w"
+     src="{{relativePath}}assets/img/image.jpg"
      alt="12 column or two up image.">
 
-<img srcset="../assets/img/image-640-bmd.jpg  640w,
-             ../assets/img/image-320-smd.jpg  320w,
-             ../assets/img/image-160-xsm.jpg  160w"
-     src="../assets/img/image.jpg"
+<img srcset="{{relativePath}}assets/img/image-640-bmd.jpg  640w,
```
Comment (paulcpederson) [inline docs/source/documentation/components/sample-code/_images.html]: this still won't work, these images need to be moved to a folder that is version controlled, yes?

**Comment by paulcpederson** [comment]
LGTM other than the images

File: docs/source/documentation/components/sample-code/_images.html (hunk 1 above)
Comment (nikolaswise) [inline docs/source/documentation/components/sample-code/_images.html]: correct

**Comment by nikolaswise** [comment]
@paulcpederson I think after this I can run release and we'll see what happens

**Pull request merged**